polaris-react/src/components/AlphaCard/AlphaCard.tsx

```diff
@@ -41,7 +41,7 @@ export const AlphaCard = ({
   roundedAbove,
 }: AlphaCardProps) => {
   const breakpoints = useBreakpoints();
-  const defaultBorderRadius = '2' as ShapeBorderRadiusScale;
+  const defaultBorderRadius = '2' as BorderRadiusScale;
```

Comment:
```suggestion
  const defaultBorderRadius: BorderRadiusScale = '2';
```

Also, can we move this value outside the body of the function?